### DISCUSSION TITLE
will node-sass dependency in nextjs be deprecated ? causes a lot of build issues on newer linux systems

### DISCUSSION THREAD

**Discussion by sandys** [vercel/next.js#29924] [Help] [answered]
hi guys,
so nextjs still has a dependency on node-sass. 
node-sass officially has declared itself deprecated - https://www.npmjs.com/package/node-sass . 

Because of this dependency, often times builds fail on newer linux machines. It needs python2 which is forcibly not made available in many contexts (especially if it is a thin docker instance).

Could you please move to just "sass" - the new officially supported library ? https://sass-lang.com/blog/libsass-is-deprecated

**Comment by timneutkens** [comment, marked as answer]
```
  "peerDependencies": {
    "fibers": ">= 3.1.0",
    "node-sass": "^4.0.0 || ^5.0.0 || ^6.0.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "sass": "^1.3.0"
  },
  "peerDependenciesMeta": {
    "node-sass": {
      "optional": true
    },
    "sass": {
      "optional": true
    },
    "fibers": {
      "optional": true
    }
  },
```

Both are optional peerdependencies. They should only be installed if your application has it in "dependencies" or "devDependencies" / if you're using an old version of npm/yarn that does not respect `peerDependenciesMeta`. We already recommend `sass` as per the docs: https://nextjs.org/docs/basic-features/built-in-css-support#sass-support

**Reply by sandys** [reply]
understood - much appreciated. Will figure out why our docker images are screwing up like this.
